README.md

```diff
@@ -1,2 +1,4 @@
 # SellWand
-A simple Sell Wand plugin. You could try it.
+A simple Sell Wand plugin. 
+# Downloads
```

Comment:
Sellwand is also available on Polymart & Songoda.

Reply:
will add those

Reply:
updated